tests/phpunit/CRM/Case/BAO/CaseTest.php

```diff
@@ -53,7 +53,7 @@ public function testCaseActivity(): void {
     $openCaseType = CRM_Core_PseudoConstant::getKey('CRM_Activity_BAO_Activity', 'activity_type_id', 'Open Case');
     while ($res->fetch()) {
       $message = 'Failed asserting that the case activity query has a activity_type_id property:';
-      $this->assertObjectHasAttribute('activity_type_id', $res, $message . PHP_EOL . print_r($res, TRUE));
+      $this->assertObjectHasProperty('activity_type_id', $res, $message . PHP_EOL . print_r($res, TRUE));
```

Comment:
It doesn't show in the jenkins output but it gives me a warning this is deprecated, so I updated it.

Reply:
Interesting - this was changed in a point release - 9.6.11, so isn't available in 9.6.5.